### PR TITLE
Added Templates support to the Config Governance SDK

### PR DESCRIPTION
**Pull request by NickXitco**
## PR summary
Added Templates support to the Config Governance SDK.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [X] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features) (API Docs PR in progress)

## Current vs new behavior  
Only current behavior being modified is patching a small bug in the Config Governance Integration tests, where running it on an account with unlabeled rules would cause a crash in the `AfterSuite()` function.

## Does this PR introduce a breaking change?    
- [ ] Yes
- [X] No

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,19 +1,16 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
-github.com/IBM/go-sdk-core/v5 v5.4.0 h1:iP6xpatSLqQ29WSZkI7x2tcNHuFds2lZhW/G+bJaj7M=
-github.com/IBM/go-sdk-core/v5 v5.4.0/go.mod h1:+MNa5Jbqb9FO7KEevo982Pb/YXr4adkyEffJlPs2TGc=
-github.com/IBM/go-sdk-core/v5 v5.4.1 h1:Q8cSK9NiXvWlUh2vy3EeTlXCAG/zCQEB+BK1jtv2ee4=
-github.com/IBM/go-sdk-core/v5 v5.4.1/go.mod h1:+MNa5Jbqb9FO7KEevo982Pb/YXr4adkyEffJlPs2TGc=
+github.com/IBM/go-sdk-core/v5 v5.4.2/go.mod h1:Sn+z+qTDREQvCr+UFa22TqqfXNxx3o723y8GsfLV8e0=
 github.com/IBM/go-sdk-core/v5 v5.4.4 h1:YgojmZYE/kaE8LxcgiRy7gm803n1z1BYeXll/gg5zuU=
 github.com/IBM/go-sdk-core/v5 v5.4.4/go.mod h1:Sn+z+qTDREQvCr+UFa22TqqfXNxx3o723y8GsfLV8e0=
+github.com/IBM/platform-services-go-sdk v0.18.18 h1:xMxM6SJH5ke04e49Mbdjmpy1po5KmvP1VE8VCzF2GAY=
```
Comment (ashishth09) [inline go.sum:5]: We shouldn't depend on platform-services-go-sdk.
Please run go mod  tidy

**Review by ashishth09 — CHANGES_REQUESTED** [review]

File: go.sum (hunk 1 above)
Comment (ashishth09) [reply go.sum:5]: Also run go fmt ./...

**Comment by gary1998** [comment]
- You need to rebase your branch with latest `main` branch. The ITs of `notificationsv1` have been fixed.

**Comment by NickXitco** [comment]
> You need to rebase your branch with latest `main` branch. The ITs of `notificationsv1` have been fixed.

Can you elaborate on this? The branch is based on the latest `main` branch as far as I can see and rebasing had no effect.

**Comment by NickXitco** [comment]
Otherwise I believe the correct changes have been made

**Comment by ashishth09** [comment]
Included in https://github.com/IBM/scc-go-sdk/pull/38 @NickXitco gave you access to create branch directly on this repo.
The integration tests required credentials available only for the pull requests from this repo.

**Pull request closed without merging**